versions.gradle

```diff
@@ -37,6 +37,7 @@ ext {
                     customtabs: "com.android.support:customtabs:$versions.supportLibrary",
                     constraint: "com.android.support.constraint:constraint-layout:1.1.0-beta4",
                     multidex  : "com.android.support:multidex:1.0.2",
+                    emoji     : "com.android.support:support-emoji:$versions.supportLibrary",
```

Comment:
`com.android.support:support-emoji-appcompat` is better cuz it's based on AppCompatTextView.